projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -84,6 +84,20 @@ export interface FeatureTogglesInterface {
    */
   productConfiguratorAttributeTypesV2?: boolean;
 
+  /**
+   * The product configuration UI is completely re-rendered after each UI interaction. This may lead to performance issues for large configuration models,
+   * where a lot of attributes (>50) and/or a lot of possible values per attribute (>50) are rendered on the UI.
+   *
+   * When this feature toggle is activated, only these parts of the UI are re-rendered, that actually changed, significantly (up to factor 10) improving rendering performance for large models.
+   *
+   * Please note, this will influence how the pricing requests are processed and rendered.
+   * Instead of merging the prices into the configuration model, which effectively triggers re-rendering the whole UI-Component tree,
+   * the price supplements are kept in a separate subtree of the model, so that the new `cx-configurator-async-pricing` component can subscribe directly to this subtree independently.
```

Comment:
@Uli-Tiger, could you check this line because `configurator-async-pricing` component does not exist?